Sources/PayPalWebPayments/PayPalWebCheckoutClient.swift

```diff
@@ -40,55 +49,66 @@ public class PayPalWebCheckoutClient: NSObject {
         analyticsService = AnalyticsService(coreConfig: config, orderID: request.orderID)
         analyticsService?.sendEvent("paypal-web-payments:checkout:started")
 
-        let baseURLString = config.environment.payPalBaseURL.absoluteString
-        let payPalCheckoutURLString =
+        Task {
+            if request.fundingSource == .card {
+                do {
+                    let configResult = try await clientConfigAPI.updateClientConfig(request: request)
+                    print("configResult: \(configResult)")
+                } catch {
+                    print("error in calling graphQL: \(error.localizedDescription)")
+                    self.notifyCheckoutFailure(with: PayPalError.payPalURLError, completion: completion)
+                }
+            }
+            let baseURLString = config.environment.payPalBaseURL.absoluteString
+            let payPalCheckoutURLString =
             "\(baseURLString)/checkoutnow?token=\(request.orderID)" +
             "&fundingSource=\(request.fundingSource.rawValue)"
```

Comment:
```suggestion
                "\(baseURLString)/checkoutnow?token=\(request.orderID)" +
                "&fundingSource=\(request.fundingSource.rawValue)"
```
quickfix: indentation.